docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: docs
+title: Command Line Interface
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+TestCafe command line interface is pretty simple.
+It consists of a single command that runs tests with a number of options.
+
+#### Synopsis
+
+```sh
+testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
+```
+
+#### Browser List
+
+Specify the list of browsers where tests will be ran.
+You can specify a browser alias or path.
```

Comment:
Need clarification about that is `alias`, need to reference `--list-browser` command here